includes/admin/tracking.php

```diff
@@ -199,10 +194,8 @@ public function check_for_optin( $data ) {
 	 */
 	public function check_for_optout( $data ) {
 
-		global $edd_options;
-		if( isset( $edd_options['allow_tracking'] ) ) {
-			unset( $edd_options['allow_tracking'] );
-			update_option( 'edd_settings', $edd_options );
+		if( edd_get_option( 'allow_tracking', false ) ) {
```

Comment:
Couldn't this just be `edd_delete_option( 'allow_tracking' );`. We're opting them out regardless if they are already opted out or not. 

If currently opted in, saves 1 db call. If not opted in neutral.